### ISSUE TITLE
Is it possible to port it to PS2?

### ISSUE THREAD

**Issue by terremoth** [protoman/rockbot#122]
It would be a nice exercise to learn about PS2 Game Development with this game.
Now https://github.com/ps2dev repos are very updated and well alive.

PS2 Forums is alive again too (https://forums.ps2dev.org/) and PS2-Scene has a Discord ( https://discord.gg/Se9W2e7 ), would be nice to see it working. So the community can maintain even more alive Open Source Console Game Development!

**Comment by protoman** [comment]
There are a few issues with the PS2 port right now.

First is performance, when I changed the game engine to run at 60 fps, the PS2 port could not reach it. I tried to add a 30 fps mode, that adjusted a multiplier to the movement speed, but it was not great. Then I've added a frameskip that seemed to work better.

But, the biggest issue was that there was not a modern gcc version so any moden-day C++ code I have, like initializing the variables in the .h file itself, did not work. There was a work in progress port of gcc 10, but the libraries I use (SDL) were not yet ported to run with that version, so that is a requirement for me to resume the port. I stopped following the project, so I don't know its current status.

If/when we have PS2SDK working with that more recent version of gcc, I'll give it a try again.

**Comment by protoman** [comment]
Oh, and you can grab the older versions of Rockbot for PS2 here:
https://rockbot.upperland.net/?page_id=301

**Comment by terremoth** [comment]
@protoman the ps2dev team ported the PS2 GCC-EE to the 10th version!!! GCC is now on 10 for PS2, you don't need to worry about this anymore. 

About the SDL, you'll have to take a look at it

**Comment by protoman** [comment]
Yeah, it is the same I already knew about. I checked the latest updates and there is still a long road ahead before I can build Rockbot with it.
